packages/backend/src/controllers/catalogController.ts

```diff
@@ -0,0 +1,52 @@
+import {
+    ApiCatalogResults,
+    ApiErrorPayload,
+    ApiExploresResults,
+} from '@lightdash/common';
+import {
+    Get,
+    Middlewares,
+    OperationId,
+    Path,
+    Request,
+    Response,
+    Route,
+    SuccessResponse,
+    Tags,
+} from '@tsoa/runtime';
+import express from 'express';
+import { allowApiKeyAuthentication, isAuthenticated } from './authentication';
+import { BaseController } from './baseController';
+
+@Route('/api/v1/projects/{projectUuid}/catalog')
+@Response<ApiErrorPayload>('default', 'Error')
+@Tags('Catalog')
+export class CatalogController extends BaseController {
+    /**
+     * Get catalog items
+     * @param projectUuid
+     * @param req contains filters for the catalog items
+     * - search: string
```

Comment:
How do you think these args would change if when we want to paginate tables or filters separately? 

Instead of onlyTable and onlyFields, what do you think of something like `tablePage` and `fieldPage`; when undefined or null, you don't get that object, but if you set it to a number you get that page? There's a few ways we could handle page size with that pattern. 

And for filters, do you think that is separate arg we add later? I imagine the form of that will be different between table/field